### PR TITLE
Rework sphinx configuration for PDF

### PR DESCRIPTION
**Pull request by FirelightFlagboy**
- Add extension `sphinx.ext.imgconverter` to allow using `svg` images in pdfs.
- Customize pdf generation:
  - `parsec.pdf`: Contains the full documentation.
  - `userguide.pdf`: Contains only the `userguide/` documentation.
  - `adminguide.pdf`: Contains only the `adminguide/` documentation.
- Include only the index note & `history` on html

  Those elements are only usefull on the website.

- Update readthedocs configuration

  - Add a dockerfile to try locally a configuration.
  - Update readthedocs with the required apt packages from the dockerfile.

Other Changes
-------------

- Remove unused `development/index.rst`

  We don't include the dev related doc in the documentation anymore. So the `index.rst` in `development` wasn't in use and sphinx generate warning for it.

- Rework `docs/Makefile`

  - Make `latexpdf` & `latexpdfja` depends on `latex`.
  - Add variable `LATEXMKOPTS` to provide the option `-interaction=nonstopmode` by default to `latexmk`.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -216,39 +223,74 @@ def fetch_parsec_version():
...
+    "fncychap": r"\usepackage[Sonny]{fncychap}",
+    "babel": r"\usepackage{babel}",
 }
 
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title, author, documentclass
 # [howto/manual]).
-latex_documents = [("index", "parsec.tex", "Parsec Documentation", "Emmanuel Leblond", "manual")]
+latex_documents = [
+    ("index", "parsec.tex", "Parsec Full Documentation", "dev-parsec+fullguide@scille.fr", "howto"),
```
Comment (mmmarcos) [inline docs/conf.py:258]:
```suggestion
    ("index", "parsec.tex", "Parsec Documentation", "dev-parsec+fullguide@scille.fr", "howto"),
```

Comment (mmmarcos) [inline docs/conf.py:258]: IMHO, `Full` is a little redundant here.
```suggestion
    ("index", "parsec.tex", "Parsec Documentation", "dev-parsec+fullguide@scille.fr", "howto"),
```

**Review by mmmarcos — APPROVED** [review]
LGTM

File: docs/conf.py (hunk 1 above)
Comment (FirelightFlagboy) [reply docs/conf.py]: done

**Pull request merged**